test/deploy.sh

```diff
@@ -136,26 +136,28 @@ NUM_NN=0
 sleep 2
 
```

Comment:
is this change related to resharding work? we don't want to effect one beacon chain